spec/features/proposal/collection_draft_proposals/collection_draft_proposal_list_spec.rb

```diff
@@ -1,47 +1,186 @@
-describe 'Viewing Unsubmitted Collection Draft Proposals', js: true do
-  before do
-    login
-    set_as_proposal_mode_mmt(with_required_acl: true)
-    4.times { create(:full_collection_draft_proposal) }
-    create(:full_collection_draft_proposal, draft_short_name: 'An Example Proposal', version: '5', draft_entry_title: 'An Example Title')
-    create(:full_collection_draft_proposal, draft_short_name: 'A Second Example Proposal', version: '')
-  end
-
-  context 'while on the manage collection proposals page' do
+describe 'Viewing Collection Draft Proposals Index Pages', js: true do
+  context 'when logged in as a user' do
     before do
-      visit manage_collection_proposals_path
+      login
+      set_as_proposal_mode_mmt(with_draft_user_acl: true)
+      4.times { create(:full_collection_draft_proposal) }
+      create(:full_collection_draft_proposal, draft_short_name: 'An Example Proposal', version: '5', draft_entry_title: 'An Example Title')
+      mock_submit(create(:full_collection_draft_proposal, draft_short_name: 'A Second Example Proposal', version: '', draft_entry_title: 'A Second Example Title', draft_request_type: 'delete'))
     end
 
-    it 'has a more button' do
-      expect(page).to have_link('More')
-    end
+    context 'while on the index page' do
+      before do
+        visit collection_draft_proposals_path
+      end
 
-    it 'displays short names correctly' do
-      expect(page).to have_content('An Example Proposal')
-      expect(page).to have_content('A Second Example Proposal')
-    end
+      it 'displays short names correctly' do
+        expect(page).to have_content('An Example Proposal')
+        expect(page).to have_content('A Second Example Proposal')
+      end
+
+      it 'displays entry titles' do
+        expect(page).to have_content('An Example Title')
+      end
+
+      it 'displays proposal status' do
+        expect(page).to have_link('In Work')
+      end
+
+      it 'displays proposal request types' do
+        expect(page).to have_content('Create')
+      end
+
+      context 'when sorting short name' do
```

Comment:
All of these sorting tests are new.  They are kind of verbose and redundant since it's the same helper functions.  I think an argument could be made that we're testing Rails, but I leave it as an open question.